inference_images.py

```diff
@@ -0,0 +1,111 @@
+import cv2
+import os
+import sys
+from mrcnn import utils
+from mrcnn import model as modellib
+from mrcnn.config import Config
+from mrcnn.visualize import *
+import mrcnn.model as modellib
+from mrcnn.model import MaskRCNN
+import uuid
+import argparse
+import skimage
+import colorsys
+import tensorflow as tf
+import numpy as np
+import shutil
+import random
+import argparse
+from skimage import io
+import uuid
+
+NUM_CATS = 46
+IMAGE_SIZE = 512
+
+
+class FashionConfig(Config):
+    NAME = "fashion"
+    GPU_COUNT = 1
+    IMAGES_PER_GPU = 1
+    NUM_CLASSES = NUM_CATS + 1
+    BACKBONE = 'resnet50'
+
+    IMAGE_MIN_DIM = IMAGE_SIZE
+    IMAGE_MAX_DIM = IMAGE_SIZE
+    IMAGE_RESIZE_MODE = 'none'
+    RPN_ANCHOR_SCALES = (16, 32, 64, 128, 256)
+
+
+config = FashionConfig()
+
+ap = argparse.ArgumentParser()
+ap.add_argument("-i", "--input", required=True)
+ap.add_argument("-o", "--output", required=True)
+args = vars(ap.parse_args())
+
+input_image_dir = args["input"]
+output_image_dir = args["output"]
+
+model_dir = '/content/drive/MyDrive/FGVC6/models/final/'
```

Comment:
Can we move this into a separate config?

Reply:
I can transfer it to the parent class